packages/internal/src/__tests__/validateSchema.test.ts

```diff
@@ -1,28 +0,0 @@
-import path from 'path'
-
-import { loadAndValidateSdls } from '../validateSchema'
-
-const FIXTURE_PATH = path.resolve(
-  __dirname,
-  '../../../../__fixtures__/example-todo-main'
-)
-
-beforeAll(() => {
-  process.env.RWJS_CWD = FIXTURE_PATH
-})
-afterAll(() => {
-  delete process.env.RWJS_CWD
-})
-
-test('Validates correctly on all platforms', async () => {
```

Comment:
We still need this test, the positive validation case that's missing in the new tests!